Report/Template.Rmd

```diff
@@ -243,17 +251,267 @@ datasets = read.and.clean(dataset = "easySHARE_rel6_0_0.rda", wav = 99)
+
+```{r, eval = FALSE}
+
+# Probit for each country and gender 
+allModels = lapply(df.splits, function(z){
+  
+  z     = z[-z$age50] # Multicollinearity
+  model = glm(z$labor_participationTRUE ~., family = binomial(link = "probit"), data = z)
+  
+  return(model)
+})
+
+# Return summaries
+allSummaries = lapply(allModels, summary)
+```
+
+
+
+The loop below is a robust way to apply the Wald Test for the dataset subsetted by country and gender and utilises only health-related variables. A list intended to log is created, which tracks failure of applying the Wald Test on joint significance of the selected variables. `Try` ensures continuation of the loop despite possible errors. In case an error occurs, the class of `testOutput` will be `try-error` and consequently logged into named list. Since it preserves the structure as specified before, results can be easily transformed and named. This setup has proven to be useful for debugging our test as well as finding out that some health variables are empty for certain subgroups, leading to selective failure of the test. 
+
+
+```{r, eval = FALSE}
+
+# Wald Test for all models
+wald.log = list() # Save Wald Test Output
+
+for(i in 1:length(allSummaries)){
+  
+  # Get Element
+  SummaryElement = allSummaries[[i]]
+  
+  # Specify the of coefficients to be tested: only health variables
+  health = c(16:19)
+  
+  # Test only the joint significance of health variables
+  testOutput = try(joint.wald.test(allSummaries[[i]], 0.95, health))
+  
+  if(class(testOutput) == "try-error"){
+    msg = paste0("Wald Test failed for Model Element ", i)
+    warning(msg)
+    wald.log[[i]] = "Error"
+  } else{
+    wald.log[[i]] = testOutput
+  }
+  rm(SummaryElement) # clean up
+}
+
+wald.bound = as.data.frame(wald.log)
+colnames(wald.bound) = names(allModels)
+
+```
+
+
+Furthermore, the application of the Wald Test can easily be substitued. We use a different Wald Test to validate our results. 
+
+```{r, eval = FALSE}
+    testOutput = try(wald.test(b = coef(ModelElement), 
+                               Sigma = vcov(ModelElement), Terms = health)$result)
+```
+
+In line with the proposed concatenated `lapply` structure, the probability of employment can be predicted for each subset. Marginal effects are calculated using `probitmfx`, which firstly recalculates a probit model. 
+
+```{r, eval = FALSE, warning = FALSE}
+# Calculate marginal effects and standard errors
+
+# Same structure as before, but must calculate model again. 
+mfx.Models = lapply(df.splits, function(z){
+    
+    z   = z[-z$age50] # Multicollinearity
+    res = probitmfx(z$labor_participationTRUE ~.,atmean = TRUE,  data = z)
+    
+    return(res)
+})
+
+```
+
+
+## Empirical Results
+
+By means of the presented model, it is evaluated how labor force participation behavior of individuals aged 50-64 is influenced by different health indicators across 11 European countries.
```

Comment:
Thats confusion. Whta is "underlying data enables us to mitigate justification bias and measurement error that occur when assessing health subjectively " supposed to mean?

Reply:
It's based upon Kalwij and Vermeulen. But the quote is missing so far. Will adjust :)

Reply:
However, it is supposed to mean that objective and subjective health often differ substantially between persons. For many studies, only subjective health is reported. Since it is subjective, it is prone to measurement errors.